scripts/gis-material-update/process/modules/autoliikennemaarat.py

```diff
@@ -48,9 +48,20 @@ def persist_to_database(self) -> None:
         )
 
         # persist traffic volume polygons to database
+        #for buffer_size, polygon_data in self._process_result.items():
```

Comment:
Why has a bunch of out-commented code been added to the process scripts?

Reply:
Actually not added. Those were already earlier in the code but now commented out because those parts are only for as local testing purposes.